submissions/xT0PERx/js-dom/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="visuality.css" />
+    <title>My interactive side-menu</title>
+  </head>
+  <body>
+    <div class="container clearfix">
```

Comment:
Add `clearfix` to the corresponding block, in your case, you don't need a `clearfix`;

Reply:
Fixed